src/Config.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
Jee, to je super tohle vyjmout ven z `run.php` :-)

Reply:
Jsem rád, že se ti to líbí. Chtěl bych ten run.php nechat úplně minimální, co to jde. 

Úplně nejradši bych tam měl i DI kontejner a nemusel si tam stavět všechny ty třídy do sebe, ale nechtěl jsem na tom zbytečně pálit moc času - nikdy jsem to bez zbytku Symfony nepoužíval.